providers/trello.js

```diff
@@ -0,0 +1,733 @@
+    }
+
+    async removeLabelFromCard(){
+        let embed = this._preparePayload();
+        embed.title = '[' + this.action.data.board.name + '] Removed Label from "' + this.action.data.card.name + '"';
+        embed.url = this._resolveFullCardURL(this.action.data.card);
+        this._formatLabel(this.action.data.text, this.action.data.value, embed);
+        this.payload.addEmbed(embed);
+    }
+
+    async removeMemberFromCard(){
+        let embed = this._preparePayload();
+        if(this.action.memberCreator.id === thos.action.member.id){
+            embed.title = '[' + this.action.data.board.name + '] Left "' + this.action.data.card.name + '"';
+        } else {
+            embed.title = '[' + this.action.data.board.name + '] Removed User from "' + this.action.data.card.name + '"';
+            Trello._addMemberThumbnail(this.action.member.avatarHash, embed);
+            embed.description = this.action.member.fullName + ' ([`' + this.action.member.username + '`](' + this.baseLink + this.action.member.username + '))';
+        }
+        embed.url = this._resolveFullCardURL(this.action.data.card);
+        this.payload.addEmbed(embed);
+    }
+
+    async removeMemberFromBoard(){
+        let embed = this._preparePayload();
+        if(this.action.memberCreator.id === this.action.member.id){
+            embed.title = 'Left Board "' + this.action.data.board.name + '"';
+        } else {
+            embed.title = 'Removed User from Board "' + this.action.data.board.name + '"';
+            embed.description = this.action.member.fullName + ' ([`' + this.action.member.username + '`](' + this.baseLink + this.action.member.username + '))';
+            Trello._addMemberThumbnail(this.action.member.avatarHash, embed);
+        }
+        embed.url = this._resolveFullBoardURL(this.action.data.board);
+        this.payload.addEmbed(embed);
+    }
+
+    async removeMemberFromOrganization(){
+        let embed = this._preparePayload();
+        if(this.action.memberCreator.id === this.action.member.id){
+            embed.title = 'Left Organization "' + this.action.data.organization.name + '"';
+        } else {
+            embed.title = 'Removed User from Organization "' + this.action.data.organization.name + '"';
+            Trello._addMemberThumbnail(this.action.member.avatarHash, embed);
+            embed.description = this.action.member.fullName + ' ([`' + this.action.member.username + '`](' + this.baseLink + this.action.member.username + '))';
+        }
+        embed.url = this._resolveGenericURL(this.action.data.organization.id);
+        this.payload.addEmbed(embed);
+    }
+
+    // How to Trigger?
+    async unconfirmedBoardInvitation(){
+        
+    }
+
+    // How to trigger?
+    async unconfirmedOrganizationInvitation(){
+        
+    }
+
+    // TODO
+    async updateBoard(){
+        //So many preferences x.x
+    }
+
+    // TODO
+    async updateCard(){
+        //So many preferences x.x
+    }
```

Comment:
The updateBoard and updateCard actions are pretty extensive. Unsure if I should go in detail with the discord embed or have a generic 'Updated Board/Updated Card' message. It's possible to do the first option, however it may take some time due to the magnitude of possibilities.